### DISCUSSION TITLE
[Script request] Lobe Chat

### DISCUSSION THREAD

**Discussion by bordeo** [community-scripts/ProxmoxVE#90] [Request script]
### Application Name

Lobe Chat

### Website

https://github.com/lobehub/lobe-chat

### Description

🤯 Lobe Chat - an open-source, modern-design AI chat framework. Supports Multi AI Providers( OpenAI / Claude 3 / Gemini / Ollama / Azure / DeepSeek), Knowledge Base (file upload / knowledge management / RAG ), Multi-Modals (Vision/TTS) and plugin system. One-click FREE deployment of your private ChatGPT/ Claude application.
Adding it to proxmox scripts provide a useful chat interface to all AI model in a private env

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
@havardthom 
Possible script for you? You have already puller the OpenWebUI. :-)
